core/flyout_horizontal.js

```diff
@@ -268,17 +270,17 @@ Blockly.HorizontalFlyout.prototype.wheel_ = function(e) {
 Blockly.HorizontalFlyout.prototype.layout_ = function(contents, gaps) {
   this.workspace_.scale = this.targetWorkspace_.scale;
   var margin = this.MARGIN;
-  var cursorX = this.RTL ? margin : margin + Blockly.BlockSvg.TAB_WIDTH;
+  var cursorX = margin + this.tabWidth_;
   var cursorY = margin;
   if (this.RTL) {
     contents = contents.reverse();
   }
 
-  for (var i = 0, item; item = contents[i]; i++) {
+  for (var i = 0, item; (item = contents[i]); i++) {
```

Comment:
Why did you add these parenthesis here and elsewhere?

Reply:
I added this updates to layout_ and reflowInternal_ because I was already looking very closely in those functions.
Adding the parenthesis prevents this line from being flagged as a warning for some debuggers and IDEs. I did a similar change in the renderers directory in #2883 where I also included some detail in the PR about the warning.